exchanges/binance/binance_live_test.go

```diff
@@ -0,0 +1,33 @@
+//+build mock_test_off
+
+// This will build if build tag mock_test_off is parsed and will do live testing
+// using all tests in (exchange)_test.go
+package binance
+
+import (
+	"os"
+	"testing"
+
+	"github.com/thrasher-corp/gocryptotrader/config"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/sharedtestvalues"
+	log "github.com/thrasher-corp/gocryptotrader/logger"
+)
+
+var mockTests = false
+
+func TestMain(m *testing.M) {
+	cfg := config.GetConfig()
+	cfg.LoadConfig("../../testdata/configtest.json")
+	binanceConfig, err := cfg.GetExchangeConfig("Binance")
+	if err != nil {
+		log.Error("Test Failed - Binance Setup() init error", err)
```

Comment:
This can be simplified to log.Fatal, as this automatically calls os.Exit(1) after printing. These will need to be updated accross the exchanges